app/src/main/java/com/x8bit/bitwarden/ui/platform/components/segment/BitwardenSegmentedButton.kt

```diff
@@ -22,21 +29,41 @@ fun BitwardenSegmentedButton(
     options: ImmutableList<SegmentedButtonState>,
     modifier: Modifier = Modifier,
 ) {
-    SingleChoiceSegmentedButtonRow(
-        modifier = modifier,
+    Box(
+        modifier = Modifier
+            .background(color = BitwardenTheme.colorScheme.background.secondary)
```

Comment:
💭 should we add in the background color as an argument? I also think I may be missing the need for the box.
would it mess things up to put the vertical padding on the row, and maybe have the composable take `backgroundColor` and `padding: PaddingValues` as arguments. I feel like the hard coded modifier with the other modifier combo is a bit confusing.